docs/TuxAnimations.md

```diff
@@ -0,0 +1,208 @@
+# Tux/Animations
+
+##Premise
+>Tux Animations is a library of default animation components and            easings. Additionally Tux Animations includes the makeAnimation function     which can be used to create custom animation components.
```

Comment:
Remove extra white space between and and easings, as well as function and which.